apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/setup-xm-cloud.md

```diff
@@ -0,0 +1,182 @@
+---
+title: 'Sprint Zero: How to Set Up and Utilize XM Cloud'
+description: 'In this blog, we will go through the steps to complete a sprint zero of setting up XM Cloud.  In addition to learning the basics of XM Cloud, you will become familiar with setting up a site, setting up the dev environment to support building that site, and how to connect your codebase that runs on your local rendering host to the preview endpoint of XM Cloud.'
+promoAfter: ['learning-essentials']
+openGraphImage: 'https://mss-p-006-delivery.stylelabs.cloud/api/public/content/1915549492604a64864578fe51d2a597?v=244f9e64'
+---
+
+## Introduction - What You are Going to Learn 
+
+In this blog, we will go through the steps to complete a sprint zero of setting up XM Cloud.  In addition to learning the basics of XM Cloud, you will become familiar with setting up a site, setting up the dev environment to support building that site, and how to connect your codebase that runs on your local rendering host to the preview endpoint of XM Cloud.  
+
+## Overview  
+
+XM Cloud focuses on being a headless, enterprise content and website management system.  You can manage XM Cloud via several UIs, and also through APIs and the CLI. For this blog, we will focus on the XM Cloud deploy UI for building and launching a site with XM Cloud. ( For more information on APIs with XM Cloud, navigate [here](https://doc.sitecore.com/xmc/en/developers/xm-cloud/xm-cloud-deploy-api.html).) 
+ 
+## Setup the project and environment  
+
+Now, we will setup the developer project and environment that we will create therein. Let’s get started! In the first stage, we will access the cloud portal and create a new *project*.  
+
+### Log into Sitecore Cloud Portal 
+
+1. To begin, open the Sitecore Cloud Portal (https://portal.sitecorecloud.io) and log in.   
+2. Access the XM Cloud project and environment management interface by clicking `XM Cloud Deploy`.  
+![Sitecore Portal - Open XM Cloud Deploy](https://sitecorecontenthub.stylelabs.cloud/api/public/content/42edc85bc7bd41e2914f29b7b260496b?v=94261954)
+
+### A Quick Dive into Projects and Environments  
```

Comment:
Relooking at this, I'd put this section before 'Login to the Sitecore cloud portal' (line 20). As I was walking through it, I had a jarring experience of starting on building the site, and then suddenly going into a broad overview.

Reply:
This seems to be resolved.